system/Database/Postgre/Connection.php

```diff
@@ -128,11 +128,14 @@ public function getVersion(): string
             return $this->dataCache['version'];
         }
 
-        if (! $this->connID || ($pgVersion = pg_version($this->connID)) === false) {
+        if (! $this->connID) {
             $this->initialize();
         }
 
-        return isset($pgVersion['server']) ? $this->dataCache['version'] = $pgVersion['server'] : false;
+        $pgVersion                  = pg_version($this->connID);
+        $this->dataCache['version'] = $pgVersion['server'] ?? '';
```

Comment:
```suggestion
        $this->dataCache['version'] = isset($pgVersion['server']) ? 
                preg_match('/^(\d+\.\d+)/', $pgVersion['server'], $matches) ? $matches[1] : '' :
                '';

```
This should work if we deal with something like `15.3 (Debian 15.3-1.pgdg110+1)` - as Kenjis pointed out earlier.

Reply:
Okay, I will take care of it in #7509